server/utils/jobjects.py

```diff
@@ -0,0 +1,11 @@
+def create_jobect(query_result):
```

Comment:
what's a jobect?

Reply:
supposed to be a short hand for job object but I spelled object incorrectly, I can change it if it's confusing

Reply:
yeah, let's just spell it out. Clarity and conciseness should work together and never in opposition to each other